### DISCUSSION TITLE
PerformConversionsJob work on Local but not in Production

### DISCUSSION THREAD

**Discussion by astrO1** [spatie/laravel-medialibrary#3749] [Q&A]
I have an application that use the last version from Spatie Media Library, but when i testing on Local enviroment all works fine, but when i testing in production sometimes work and sometimes don't, i'm still getting this error on failed_jobs table

**ErrorException: unserialize(): Error at offset 2375 of 10627 bytes in**

I think is some caching issue or something else.

Anyone already have this problem?

My configuration from PHP is all fine upload_max_filesize and etc

**Comment by niravswami** [comment]
Same issue here. does anyone solved it?
